client/webserver/site/src/js/order.ts

```diff
@@ -104,7 +107,11 @@ export default class OrderPage extends BasePage {
       match: (note: MatchNote) => { this.handleMatchNote(note) }
     })
 
-    this.start()
+    this.start().then(
+      () => {
+        this.showMatchCards()
+      }
+    )
```

Comment:
Why not just put `this.showMatchCards()` at the end of `start`?

Reply:
True